npc/re/warps/cities/izlude.txt

```diff
@@ -52,6 +52,68 @@ izlude_b,52,172,0	warp	iz04_b	2,2,izlude_in,74,161
 izlude_c,52,172,0	warp	iz04_c	2,2,izlude_in,74,161
 izlude_d,52,172,0	warp	iz04_d	2,2,izlude_in,74,161
 
+//= iz_int(Intro) & int_land(Secluded Island) ================
+iz_int,27,30,0	script	iz_int#room1-1	WARPNPC,2,2,{
+	end;
+
+OnTouch:
+	warp "iz_int", 51, 30;
+	end;
+
+OnInit:
+	questinfo 21001, QTYPE_QUEST, 1, Job_Novice;
+	end;
+}
+
+iz_int,47,30,3	script	iz_int#room1-2	WARPNPC,2,2,{
+	end;
+
+OnTouch:
+	@display = 0;
+	warp "iz_int", 22, 30;
+	end;
+}
+
+iz_int,56,15,0	script	iz_int	WARPNPC,2,2,{
+	end;
+
+OnTouch:
+	savepoint "int_land", 77, 101;
+	warp "int_land", 85, 107;
+	end;
+}
+
+int_land,49,57,0	script	noviship#izlude	WARPNPC,2,2,{
+	end;
+
+OnTouch:
+	mes "^4d4dffOnce you left this island there is no way back.";
+	mes "Are you sure you want to go directly to Izlude?^000000";
+	next;
+	mes "^4d4dffIf you do, the quest will be deleted from your Quest Log.^000000";
+	next;
+	if (select("Do not go to Izlude yet", "Sail to Izlude!") == 1) {
+		mes "[Sailor]";
+		mes "You want to stay here longer?";
+		mes "Good, then I'll wait until you're ready~";
+		close;
+	}
+	mes "[Sailor]";
+	mes "Lets head toward Izlude!";
+	close2;
+	if (checkquest(21001))
+		erasequest(21001);
+	if (checkquest(21002))
+		erasequest(21002);
+
+	.@map$ = "izlude";
```

Comment:
I also would like to check if the `npctalk` and `classchange` are attached to the player (can't be see by other player?).